Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.17.7-13.1655148239 as builder
+FROM golang:1.18 as builder
```

Comment:
@gabemontero do you know if we have an official go 1.18 image anywhere? Most projects seem to use this one but dockerhub images can run into rate limiting issues.